lib/stm32/f7/Makefile

```diff
@@ -40,9 +40,9 @@ TGT_CFLAGS	= -Os -g \
 
 ARFLAGS		= rcs
 
-OBJS		= rcc.o gpio.o gpio_common_all.o gpio_common_f0234.o
+OBJS		= flash.o rcc.o gpio.o gpio_common_all.o gpio_common_f0234.o
 
-OBJS		+= rcc_common_all.o flash_common_f234.o flash_common_f24.o
```

Comment:
There's _nothing_ in common with flash f234/f24 anymore?  I find that a little hard to believe.